docs/configuration/recording_rules.md

```diff
@@ -86,6 +86,9 @@ name: <string>
 # rule can produce. 0 is no limit.
 [ limit: <int> | default = 0 ]
 
+# Offset the rule evaluation of this particular group by the specified duration.
```

Comment:
I use slightly different wording here as this is the most specific one - instead of saying "initial delay.." I word it in the context of the rule groups.

Reply:
```suggestion
# Offset the rule evaluation timestamp of this particular group by the specified duration into the past.
```

Reply:
(as mentioned above, I'd also go away from the "Initial delay..." wording elsewhere, as it was a bit misleading)